cmd/apikey/apikey_save.go

```diff
@@ -132,6 +133,27 @@ var apikeySaveCmd = &cobra.Command{
 			config.SaveConfig()
 		}
 
+		if config.Current.Clusters == nil {
```

Comment:
This logic should be in the config.go file while CLI is reading/loading the config